src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -91,6 +91,10 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
 
     if (!wallet) throw new Error('No wallet connected')
 
+    if(Number(values.cryptoAmount) === 0){
+      values.cryptoAmount="0.00001"
+    }
```

Comment:
I don't think this is a practical solution. It's going to make everything inaccurate, and isn't guaranteed to make the fee estimation correct. A better solution (though still not good) would be adding a fixed fudge-factor to the gasLimit.

Reply:
yeah sorry @ErvinDimitri this is not a viable solution

Reply:
I performed some tests and concluded that `getFeeData()` is working as expected.
Here is a different approach, to estimate a Gas Limit, only when the input value is greater than 0
Please check the PR: https://github.com/shapeshift/web/pull/1215